python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -42,6 +43,12 @@
 def load_config():
     """Load the Ganga config files before the test and clean them up afterwards"""
     load_config_files()
+
+    # make sure post-boostrap hook is run to ensure Dirac config options are set correctly
+    # Only becomes an issue if this test is run on it's own
+    from GangaLHCb import postBootstrapHook
+    postBootstrapHook()
+
```

Comment:
Without this, the appropriate DIRAC config variables aren't setup correctly for a unit test on it's own